apps/orams/components/CreatePassword/CreatePassword.js

```diff
@@ -0,0 +1,112 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import { Form } from 'react-redux-form'
+import PageAlert from '@gov.au/page-alerts'
+
+import ErrorBox from 'shared/form/ErrorBox'
+import Textfield from 'shared/form/Textfield'
+import CheckboxDetailsField from 'shared/form/CheckboxDetailsField'
+import { passwordLength } from 'shared/validators'
+import LoadingButton from 'shared/LoadingButton/LoadingButton'
+
+import styles from './CreatePassword.scss'
+
+const Signup = props => {
+  const { model, submitClicked, handleSubmit, currentlySending, createUserSuccess } = props
+
+  let hasFocused = false
+  const setFocus = e => {
+    if (!hasFocused) {
+      hasFocused = true
+      e.focus()
+    }
+  }
+
+  return (
+    <main>
```

Comment:
when  the token is still being decoded, shouldn't the user see a "loading" indicator?

Reply:
Ok put a loading indicator. If the token is invalid... mmmh it shouldn't be invalid :P